### PR TITLE
Popup modal

### PR DESCRIPTION
**Pull request by MohamedHNoor**
### Created popup modal:

- using basic JavaScript syntax.
- used JavaScript to manipulate DOM elements.
- used the Figma design to create the project feature.
- Used objects to store and access data.
-  created HTML elements inside javascript

### CODE REVIEW

File: js/main.js [hunk 1]
```diff
@@ -10,3 +10,85 @@ navLinks.forEach((link) => {
...
+    "CodeKit",
+    "GitHub",
+    "JavaScript",
+    "Bootstrap",
+    "Terminal",
+    "Codepen",
+  ],
+  sourceLink: "https://github.com/MohamedHNoor/mobile-firs-project1",
+  liveLink: "https://mohamedhnoor.github.io/Portfolio-project1/",
+};
```
Comment (tresorsawasawa) [inline js/main.js]: - [ ] Kindly make sure the JavaScript code implements an array of objects to store JavaScript objects containing the information for each project and not only for one project. i.e. you should repeat the same approach as you did above for each project object and all objects should be wrapped in an `Array`.
e.g.:
```javascript
const projects = [
  {
    id: "1",
    name: "Submarine game",
    description: "A Phaser3 platform game built as part of the Microverse JavaScript program.",
    image: "link_to_image.jpg",
    technologies: ["Phaser3", "JavaScript", "Jest"],
    link: "link_to_live_version.com",
    link: "link_to_source_version.com",
},
 {
    id: "2",
    name: "Submarine game",
    description: "A Phaser3 platform game built as part of the Microverse JavaScript program.",
    image: "link_to_image.jpg",
    technologies: ["Phaser3", "JavaScript", "Jest"],
    link: "link_to_live_version.com",
    link: "link_to_source_version.com",
},
...
]
```

File: index.html [hunk 2]
```diff
@@ -83,6 +84,9 @@ <h1 class="headline-title">
       </a>
     </section>
 
+    <!-- The Modal -->
+    <section id="myModal" class="modal"></section>
+
     <!-- work section -->
     <section id="portfolio" class="work-container container">
       <div class="work-heading">
```
Comment (tresorsawasawa) [inline index.html:92]: - [ ] Kindly make sure the **work section** is created dynamically with the information from the projects array(all the HTML of the section is created when the page loads). You should remove all the created HTML code for the project's list, and instead, it should be added via JavaScript. Here is an **[example](https://github.com/olawale-o/olawale-o.me/pull/15/files#diff-0eb547304658805aad788d320f10bf1f292797b5e6d745a3bf617584da017051R95)** for more details.

- [ ] Once you implement the above changes in your `.js` file you will make sure that your **Modal popup** data is populated with the data from the projects array you created previously.

File: css/modal.css [hunk 3]
```diff
@@ -0,0 +1,205 @@
...
+  line-height: 16px;
+  letter-spacing: 0.03em;
+  color: #3a4a42;
+}
+
+#description {
+  font-family: "Inter", sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 18px;
```
Comment (tresorsawasawa) [inline css/modal.css:99]: - [ ] Kindly make sure you reduce the **font size** of the project description text and use the one from your template to match your chosen Figma design template

File: css/modal.css [hunk 4]
```diff
@@ -0,0 +1,205 @@
...
+  padding-top: 100px;
+  padding-bottom: 100px;
+  left: 0;
+  top: 0;
+  width: 100%;
+  height: 100%;
+  overflow: auto;
+  background-color: rgb(0, 0, 0);
+  background-color: rgba(0, 0, 0, 0.4);
+}
```
Comment (tresorsawasawa) [inline css/modal.css]: - [ ] Kindly make sure you add the **blur** effect on the background color of your **Modal Popup** to match the Figma design.
#### Your Background view
Screenshot from 2022-09-07 15-22-24

#### Figma view
Screenshot from 2022-09-07 15-23-54

File: css/modal.css [hunk 5]
```diff
@@ -0,0 +1,205 @@
...
+
+/* The Close Button */
+.close {
+  color: #aaa;
+  float: right;
+  font-size: 28px;
+  font-weight: bold;
+}
+
```
Comment (tresorsawasawa) [inline css/modal.css]: - [ ] Kindly make sure you update the styling of your **Close Modal Button** to match the one from the Figma template

| Figma Design | Your Design |
| --------------------- | ----------------- |
| Screenshot from 2022-09-07 15-26-18 | Screenshot from 2022-09-07 15-26-37 |

Comment (tresorsawasawa) [inline css/modal.css]: - [ ] Kindly make sure you fix the modal styling in some screen sizes as your Modal Popup is supposed to be responsive. As you can see in the image below the **See Live** and **See Source** buttons are overlapping over some text.
Screenshot from 2022-09-07 15-33-04

**Review by tresorsawasawa — CHANGES_REQUESTED** [review]
# STATUS: REQUIRED CHANGES :recycle::recycle: 

Hi @MohamedHNoor 
Good job so far! I know you worked hard to implement this project, I really appreciate the effort you put in! But there are some issues that you still need to work on to go to the next project but you are almost there! 

almost-there

## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- Good commit history :heavy_check_mark: 
- Good PR description :heavy_check_mark: 
- Follow GitHub flow :heavy_check_mark: 

## Required changes :recycle: :recycle: 
- [ ] **Kindly make sure you update your linters configuration, right now you are using the wrong ones. I suggest you [click here](https://github.com/microverseinc/linters-config/tree/master/html-css-js) get the required linters for this project.**
- [ ] Check the comments under the review.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

_Feel free to leave any questions or comments in the PR thread if something is not 100% clear. 
Please ping me [@tresorsawasawa](https://github.com/tresorsawasawa) when you comment so I can receive the notification or use slack with the same name._

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by ShahierNashaat — APPROVED** [review]
Hi @MohamedHNoor, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## To Highlight 💯 

- Correct use of GitFlow ✔️
- Descriptive commit messages ✔️
- Professional looking pull request ✔️
- Your readme looks professional ✔️
- Good overall design ✔️
- Your code is clean and well documented ✔️

# Approved :ok_hand: :ok_hand: :ok_hand:

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**